src/getPage.tsx

```diff
@@ -40,7 +41,12 @@ export default async function getPage({
   router = (router) => router,
   useApp = true,
   useDocument = false,
-}: Options): Promise<{ page: React.ReactElement; html: string }> {
+}: Options): Promise<
+  {
```

Comment:
Let's extract this into a type (e.g. `GetPageResult`)  and export it from library as well? It might be useful for people trying to wrap "getPage" with some default options.

Reply:
You mean to avoid typing:

```js
import { getPage } from 'next-page-tester';
type GetPageResult = ReturnType<typeof getPage>
```
?

Reply:
Yeah. I know this is possible, but most libraries import this for ease of usage.

e.g.

```js
import { RenderResult } from '@testing-library/react';
```